cli/tests/snapshot/main.rs

```diff
@@ -43,10 +43,24 @@ fn check_snapshots(path: &str) {
 
     // We set the `-f somefile.ncl` argument before the others, because it can't come after
     // customize mode's delimiter `--`.
-    let invocation = NickelInvocation::new()
-        .file(&file)
-        .args(annotation.command)
-        .extra_args(annotation.extra_args);
+    let invocation = if annotation.is_package_manifest {
+        #[cfg(not(feature = "package-experimental"))]
+        return;
+
+        #[allow(unreachable_code)]
```

Comment:
This annotation is a bit surprising, semantically speaking. I wonder if using something like `#[cfg(feature = "package-experimental")]` would better convey the meaning.

Reply:
@jneem now you should be able to get rid of the `[allow(unreachable_code]]`, don't you?